doc/user-guides/authenticated-rl-with-jwt-and-k8s-authnz.md

```diff
@@ -49,12 +41,6 @@ Create the `HTTPRoute`:
 kubectl apply -f examples/toystore/httproute.yaml
 ```
 
-Expose the API:
```

Comment:
this is needed when not running in Linux

Reply:
On macOS at least, it's not needed anymore.

Reply:
I've added the following note a bit further down the guide instead:

```
It should return `200 OK`.

**Note**: This only works out of the box on linux environments. If not on linux,
you may need to forward ports

kubectl port-forward -n istio-system service/istio-ingressgateway 9080:80 &
```

This keeps it consistent with the other user guides which specifies this note instead of explicity requiring the user to port forward as a guide step 🤔